src/js/components/search/filters/location/CountryType.jsx

```diff
@@ -6,39 +6,55 @@
 import React from 'react';
 
 const propTypes = {
-    toggleCountry: React.PropTypes.func
+    toggleCountry: React.PropTypes.func,
+    locationOption: React.PropTypes.string,
+    countries: React.PropTypes.array
 };
 
-export default class CountryType extends React.Component {
+const defaultProps = {
+    countries: [
+        "All",
+        "USA",
+        "Foreign"
+    ]
+};
 
+export default class CountryType extends React.Component {
     render() {
+        const countries =
+            this.props.countries.map((name) => {
```

Comment:
This is getting close, but `defaultChecked` is generally a sign that a component is uncontrolled. Here's an example of what a controlled component looks like: [http://codepen.io/anon/pen/JbVdxB](http://codepen.io/anon/pen/JbVdxB)

Basically a component is controlled if I can update its UI by modifying a parent's state via the Redux dev tools instead of clicking. I'm pretty sure this set up actually passes that test, but `defaultChecked` does not force a checked state, which the codepen above does.